steely/plugins/b.py

```diff
@@ -0,0 +1,31 @@
+COMMAND = '.b'
+__author__ = ('byxor',)
+
+
+def main(bot, author_id, message, thread_id, thread_type, **kwargs):
+    last_message = lambda: bot.fetchThreadMessages(thread_id=thread_id, limit=2)[1]
+    send = lambda message: bot.sendMessage(message, thread_id=thread_id, thread_type=thread_type)
+
+    send(encode(last_message()))
+
+    
+REPLACEABLE = [character for character in 'PpBbGg']
+REPLACEMENT = '🅱️'
+    
+
+def encode(string):
+    for sequence in REPLACEABLE:
+        string = string.replace(sequence, REPLACEMENT)
+    return string
+
+
+def help():
+
+    def code_block(contents):
+        tag = "```"
+        return tag + contents + tag
```

Comment:
does this need to be encapsulated? you just trying to put it in a nice place?

Reply:
It doesn't need to be; I just like defining the knowledge for something in 1 place.

I actually plan to move the formatting helpers out to a little helper module soon for one of my next hacktoberfest spam PRs.

e.g.
```
formatting module = {
  bold: ...,
  italic: ...,
  strikethrough: ...,
  code_block: ...,
  monospace: ...,
}
```